packages/dx-react-scheduler/docs/reference/grouping-state.md

```diff
@@ -21,6 +21,7 @@ None
 Name | Type | Default | Description
 -----|------|---------|------------
 grouping? | Array&lt;[Grouping](#grouping)&gt; | | Specifies resources to group by.
+groupByDate? | (viewName: string) => boolean | `() => false` | Specifies whether appointments should be grouped by date in a specific view.
```

Comment:
`groupByDate` sounds more like an action than query. We should consider name like `isGroupByDate` or similar